data/uhc_pack/functions/lobby/randomize_teams.mcfunction

```diff
@@ -3,7 +3,7 @@
 # Tail recurse if there are still more players at the end
 #
 
-execute if score UHC uhcTeams matches 2.. run team join redstone @r[team=]
```

Comment:
This is actually intentional. There is minimum of 2 teams for a UHC, so it always randomizes into at least 2 teams